docker/proxy-nginx.Dockerfile

```diff
@@ -15,3 +17,7 @@ RUN  grep -P '^([A-Z_]*_PORT[A-Z_]*)\s*=\s*([0-9]){4,5}$' /root/address.py \
      | while IFS=$' \t=' read VAR VALUE; do \
          sed -i "s/{$VAR}/$VALUE/g" /etc/nginx/nginx.conf;\
      done
+
+RUN a=`getent hosts host.docker.internal | awk '{ print $1 }'` \
+    && [[ -n "$a" ]] && HOST_ADDR=$a || HOST_ADDR="172.19.0.1" \
```

Comment:
The default address here is sort-of dependent on `docker/docker-compose.yml` (specifically, the `networks` section). Perhaps it's worth mentioning that in a comment here? In general, I think this entire `RUN` command could use a comment to describe why it's necessary.